src/js/OTSubscriber.coffee

```diff
@@ -4,7 +4,7 @@
 #     stream (Stream) - stream to which you are subscribing
 #   Methods: 
 #     getAudioVolume()
-#     getImgData() : String
+#     getImgData(successCallback, errorCallback)
```

Comment:
@ggoldens Shouldn't we follow JS practices and do `getImgData(error, success)` ? 

This way we can do something like this:
```
publisher.getImgData(function(error, data) {
  if (error) {
    console.log('error', error):
  } else {
    // do something with the data
  }
});